yaptide/persistence/db_methods.py

```diff
@@ -58,6 +54,12 @@ def fetch_simulation_by_job_id(job_id: str) -> Union[BatchSimulationModel, Celer
     return simulation
 
 
+def fetch_simulation_id_by_job_id(job_id: str) -> int:
+    """Fetches simulation by job id"""
+    simulation_id = db.session.query(SimulationModel.id).filter_by(job_id=job_id).first()[0]
+    return simulation_id
```

Comment:
How this will behave if there is no such simulation ? Will an exception be thrown ?

Another things - explain in the comments (docstring) how this method differs from `fetch_celery_simulation_by_job_id`

Reply:
It was throwing exception. I have made changes so that None will be returned - same as for other similar  methods